lumeer-storage/lumeer-storage-mongodb/src/main/java/io/lumeer/storage/mongodb/dao/project/MorphiaCollectionDao.java

```diff
@@ -92,6 +92,13 @@ public Collection getCollectionByCode(final String code) {
       return collection;
    }
 
+   @Override
+   public List<Collection> getCollectionByIds(final List<String> ids) {
```

Comment:
What about using `Collection<String>` here? I can see that we sometimes need to pass `Set` here and it doesn't really matter what collection type it is inside this method.